### PR TITLE
a Tuto about how to make a DynDNS with DNSimple

### PR DESCRIPTION
**Pull request by jodumont**

### CODE REVIEW

File: exoscale_documentation/dns/40_ddns_api-recipes.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+1. **Make script file like /root/bin/update_exodns.sh**
+cut and past this inside this file
+*find your API Key and Secret [here](https://portal.exoscale.ch/account/profile/api)*
+
+		#!/bin/bash
+		LOGIN="___PUT YOUR API Key ___"
+		TOKEN="___PUT YOUR Secret Key___"
+		DOMAIN_ID="___PUT YOUR DOMAIN NAME___"
+		IP=`curl -s http://icanhazip.com/`
```
Comment (marcaurele) [inline exoscale_documentation/dns/40_ddns_api-recipes.md]: To not rely on a web host, you can run `dig +short myip.opendns.com @resolver1.opendns.com` which will retrieve your IP address.

**Comment by matthewrevell** [comment]
Hey!

Thanks so much for this! I'm sorry for the delay in replying.

I'd like to propose a couple of edits, and I'll do so soon. Then, if you're happy with what I suggest, we can get this merged in.

Cheers!

Matthew.

File: exoscale_documentation/dns/40_ddns_api-recipes.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+title: "Make your own DynDNS"
```
Comment (matthewrevell) [inline exoscale_documentation/dns/40_ddns_api-recipes.md]: I think we should replace DynDNS with "dynamic DNS service" so we avoid confusion.

File: exoscale_documentation/dns/40_ddns_api-recipes.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+date: "2016-04-28 00:00 +01:00"
+authors: "JOduMonT"
+category: "dns"
+slug: "dns"
+tags: "dns, dyndns, api, script, tutorials"
+---
+
+# Make your own DynDNS with Exoscale
+
+## Prerequisite
```
Comment (matthewrevell) [inline exoscale_documentation/dns/40_ddns_api-recipes.md]: Prerequisites

File: exoscale_documentation/dns/40_ddns_api-recipes.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+1. Having a DNS Zone
+2. Created the subdomain you want to point to your dynamic IP
+
+##1. Throught the Exoscale portal
```
Comment (matthewrevell) [inline exoscale_documentation/dns/40_ddns_api-recipes.md]: s/Throught/In

**Comment by matthewrevell** [comment]
Hi, any thoughts on the changes I've suggested so far? I have more but would like to check in with you before I go ahead.

**Comment by pyr** [comment]
@matthewrevell I'm fine with the edits. Since you're clearly the authority on that subject I suggest that you directly push your edits to the branch. I think it's a rare case where you'll always win by argument through authority.

**Comment by matthewrevell** [comment]
I've made substantial changes to the original and I've created a new PR here: https://github.com/exoscale/tutorials/pull/18

**Comment by ste-m** [comment]
Hi,
@ste-m from Exoscale here.
I'm sorry to communicate that we decided to close the contribution project.
We thank you for the time you've spent on this PR but we will now close it.

**Pull request closed without merging**